packages/devtools_extensions/example/foo_devtools_extension/lib/main.dart

```diff
@@ -74,6 +74,29 @@ class _FooExtensionHomePageState extends State<FooExtensionHomePage> {
             ),
             const SizedBox(height: 48.0),
             Text('Received message from DevTools: $_message'),
+            const SizedBox(height: 48.0),
```

Comment:
Nit: `const SizedBox(height: 64.0)` instead of two `SizedBox`es?

Reply:
removed